extensions/shared/external-media/sources/google-photos/google-photos-media.js

```diff
@@ -77,17 +78,26 @@ function GooglePhotosMedia( props ) {
 		}
 	}, [ lastQuery, listUrl, getNextPage, path ] );
 
+	const viewSelectFormEl = useRef( null );
+	useEffect( () => {
+		const viewSelectEl = viewSelectFormEl.current.elements[ 0 ];
```

Comment:
I think we could rename this to firstFormEl or the like. In this context we don't really care _which_ input we're focusing as long as it's the first one.

Reply:
I decided to drop the autofocus part here in https://github.com/Automattic/jetpack/pull/15868/commits/0062aecaa7dcb30445a6f55c2426529de25a5068 (see commit description for more info!).